code/src/index.css

```diff
@@ -1,13 +1,69 @@
+* {
+	box-sizing: border-box;
+  }
+
+  @import url('https://fonts.googleapis.com/css2?family=Oxygen+Mono&family=Roboto:wght@300&display=swap');
+
 body {
-  margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
+	font-family: 'Oxygen Mono', monospace;
+}
+
+
+
+.bigWrapper {
+	display: flex;
+    flex-direction: column;
+	max-width: 400px;
+	width: 90%;
+	margin: 10% auto;
+    margin-top: 5%;
+}
+
+button {
+	border: none;
+}
+
+.loading-overlay {
+	position: absolute;
+	top: 0;
+	right: 0;
+	bottom: 0;
+	left: 0;
+	background: #fff;
+	display: flex;
+	justify-content: center;
+	align-items: center;
+}
+
+.loading-spinner {
+	width: 50px;
+	height: 50px;
+	border: 4px solid #ffadad;
+	border-radius: 50%;
+	border-left: 5px solid #fff;
+	animation: spinner infinite 0.3s;
```

Comment:
I like that your spinner were so smooth! I guess it is because you set a faster animation